draft-ietf-rtcweb-jsep.xml

```diff
@@ -5832,8 +5832,10 @@ a=rtcp-fb:100 nack pli
           "m=" line for that MID.</t>
 
           <t>If the packet's SSRC is in the incoming SSRC mapping
-          table, route the packet to the associated "m=" line and
-          stop.</t>
+          table, check that the packet's PT matches a PT included
+          on the associated "m=" line.  If so, route the packet to
+          that associated "m=" line and stop, otherwise drop the
```

Comment:
I don't think we want this check here.  I think we we want to route the packet by SSRC and then have the RtpTransceiver drop the packet because the PT doesn't match.  If we do this check here, we end up with a strange edge case where we receive a packet with the SSRC of RtpTransceiver A and PT or RtpTransceiver B, and we end up routing it to B.  I think we should route it to A and then have A drop it.   Anyone sending such a packet is in error and it's better to fail to decode that to route media in a surprising way.